docs/web-vitals.md

```diff
@@ -0,0 +1,295 @@
+# Web Vitals
+
+By default, Honeycomb includes events for all web vitals except [first-input-delay](https://web.dev/articles/fid). First-input-delay is set to be replaced by interaction-to-next-paint as a core web vital in March 2024.
+
+## Initialize Instrumentation
+
+To initialize this instrumentation, add it to the list of instrumentations.
+
+```js
+import { HoneycombWebSDK, WebVitalsInstrumentation } from '@honeycombio/opentelemetry-web'
+
+const sdk = new HoneycombWebSDK({
+  serviceName: 'my-app',
+  instrumentations: [new WebVitalsInstrumentation()],
+});
+```
+
+## Vitals Attributes
+
+All vitals have the following attributes, they will each be namespaced by the name of the vital:
+
+```ts
+/**
+ * The name of the vital (in acronym form).
+ */
+"<vital-name>.name": 'CLS' | 'FCP' | 'INP' | 'LCP' | 'TTFB';
```

Comment:
because `name` is also getting set to the vital acronym, i do have an itch to keep this under the web_vital namespace, or scrap it all together.

thinking either:
```
name: "CLS"
web_vital.name: "CLS"
```

or 

```
name: CLS
```

alternatively, using `<vital-name>.name' to display the long-form of the acronym:

```
name: "CLS"
cls.name: "cumulative-layout-shift"
```

Reply:
I agree, let's send only `name` for now!